gorm/fields.go

```diff
@@ -3,37 +3,39 @@ package gorm
 import (
 	"context"
 	"fmt"
+	"github.com/infobloxopen/atlas-app-toolkit/util"
```

Comment:
please revert the change in import order. usually you can run `go fmt ./...` in the parent directory and it will sort these for you